src/main/scala/com/ibm/couchdb/api/Documents.scala

```diff
@@ -19,7 +19,7 @@ package com.ibm.couchdb.api
 import java.nio.file.{Files, Paths}
 
 import com.ibm.couchdb._
-import com.ibm.couchdb.api.builders.{GetDocumentQueryBuilder, GetManyDocumentsQueryBuilder}
+import com.ibm.couchdb.api.builders._
```

Comment:
was a bit too long (and wrapped) when we had a third import for the allowmissing builder, make sense to change back now